eventlogger/src/test/resources/application.properties

```diff
@@ -16,6 +16,8 @@ efs.mount=${java.io.tmpdir}/jobdownloads/
 logging.level.liquibase=ERROR
 
 execution.env=${AB2D_EXECUTION_ENV:#{'dev'}}
+# Default to true for testing
+eventlogger.kinesis.enabled=send_events
```

Comment:
Does the case matter? send_events vs SEND_EVENTS?

Reply:
@sharshar-sb it does not matter actually I tested with both cases.